src/selectors/entities/sidebar.js

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {createSelector} from 'reselect';
+
+import {General, Preferences} from 'constants/index';
+import {getConfig} from 'selectors/entities/general';
+import {getBool as getBoolPreference, get as getPreference} from 'selectors/entities/preferences';
+
+const defaultSidebarPrefs = {
+    grouping: 'by_type',
+    unreads_at_top: 'true',
+    favorite_at_top: 'true',
+    sorting: 'alpha',
+};
+
+export const getSidebarPreferences = createSelector(
```

Comment:
Could you move this into `selectors/entities/preferences` since `sidebar` isn't an entity?